### ISSUE TITLE
Consider adding older engine versions (like 7.2) 

### ISSUE THREAD

**Issue by roshkhatri** [valkey-io/valkey-json#5]
This can help in catching regressions.https://github.com/valkey-io/valkeyJSON/pull/1#discussion_r1859494869

**Comment by joehu21** [comment]
Agreed we should use the tip of 8.0 and support 7.2. Should we also support 7.0 and 6.2?

**Comment by roshkhatri** [comment]
I think we should only support 7.2 and above as valkey does not have releases for pre 7.2 version.

**Comment by joehu21** [comment]
The issue depends on this backport PR: https://github.com/valkey-io/valkey/pull/1413

**Comment by roshkhatri** [comment]
Adding support for older version will not be possible as older versions like 7.2 does not support registering new ACL categories for modules.
